src/scorepy/events.cpp

```diff
@@ -37,49 +37,60 @@ static const std::array<std::string, 2> EXIT_REGION_WHITELIST = {
 #endif
 };
 
+static region_handle create_user_region(const std::string& region_name, const CString module,
```

Comment:
Because this is what you have when calling the function. I could add an overloaded constructor to `CString` so it can be implicitely (or better explicitely) converted from a `std::string` if you like.